backend/src/question_service/models/question.ts

```diff
@@ -0,0 +1,40 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const complexityEnum = ["Easy", "Medium", "Hard"];
+
+const questionSchema = new Schema({
+  questionID: {
+    type: Number,
+    required: true,
+    unique: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  categories: {
+    type: [String],
+    required: true,
+  },
+  complexity: {
+    type: String,
+    enum: complexityEnum,
+    required: true,
+  },
+  linkToQuestion: {
+    type: String,
+    required: true,
+  },
+  questionDescription: {
+    type: String,
+    required: true,
+  },
+  questionImageURL: {
```

Comment:
question: what would this store? and if its the images in the description - what happens if there is more than one? 🤔

Reply:
for images, do you think we should just store the url since we're probably going to retrieve our questions from some api like leetcode, where we can scrape the image urls instead of storing them as actual images

i agree with handling multiple images, not many but there are questions with multiple images, [example question](https://leetcode.com/problems/reverse-linked-list/), so i change it to a string array?